pkg/restapi/issuer/operation/operations.go

```diff
@@ -366,6 +371,25 @@ func (c *Operation) registerHandler() {
 	}
 }
 
+// getProfile matches selected profile with the profiles loaded from the profiles mapping.
+// Returns a profile and an operation status.
+func (c *Operation) getProfile(w http.ResponseWriter, r *http.Request) (Profile, bool) {
```

Comment:
Can simplify the return to (*Profile) and also add a comment explaining the response you write (or break up the function per our discussion)

Reply:
Addressed